client/src/app/site/motions/services/motion-repository.service.ts

```diff
@@ -370,4 +385,79 @@ export class MotionRepositoryService extends BaseRepository<ViewMotion, Motion>
 
         return diff;
     }
+
+    /**
+     * Given an amendments, this returns the motion affected by this amendments
+     *
+     * @param amendment
+     */
+    public getAmendmentBaseMotion(amendment: ViewMotion): ViewMotion {
+        return this.getViewModel(amendment.parent_id);
+    }
+
+    /**
+     * Splits a motion into paragraphs, optionally adding line numbers
+     *
+     * @param motion
+     * @param lineBreaks
+     */
+    public getTextParagraphs(motion: ViewMotion, lineBreaks: boolean): string[] {
+        if (!motion) {
+            return [];
+        }
+        let html = motion.text;
+        if (lineBreaks) {
+            const lineLength = motion.lineLength;
+            html = this.lineNumbering.insertLineNumbers(html, lineLength);
+        }
+        return this.lineNumbering.splitToParagraphs(html);
+    }
+
+    /**
+     * Returns all paragraphs that are affected by the given amendment in diff-format
+     *
+     * @param amendment
+     */
+    public getAmendedParagraphs(amendment: ViewMotion): DiffLinesInParagraph[] {
+        const motion = this.getAmendmentBaseMotion(amendment),
+            baseParagraphs = this.getTextParagraphs(motion, true),
```

Comment:
please use `const` all the time when declaring variables. Also in the other functions.